l1-contracts/contracts/governance/IChainAdmin.sol

```diff
@@ -2,36 +2,37 @@
 
 pragma solidity 0.8.24;
 
-import {IAdmin} from "../state-transition/chain-interfaces/IAdmin.sol";
+import {Call} from "./Common.sol";
 
 /// @title ChainAdmin contract interface
 /// @author Matter Labs
 /// @custom:security-contact security@matterlabs.dev
 interface IChainAdmin {
-    /// @dev Represents a call to be made during multicall.
-    /// @param target The address to which the call will be made.
-    /// @param value The amount of Ether (in wei) to be sent along with the call.
-    /// @param data The calldata to be executed on the `target` address.
-    struct Call {
-        address target;
-        uint256 value;
-        bytes data;
-    }
-
     /// @notice Emitted when the expected upgrade timestamp for a specific protocol version is set.
-    event UpdateUpgradeTimestamp(uint256 indexed _protocolVersion, uint256 _upgradeTimestamp);
+    event UpdateUpgradeTimestamp(uint256 indexed protocolVersion, uint256 _upgradeTimestamp);
```

Comment:
Why?

Reply:
because in events' signatures we never use `_`